poetry.lock

```diff
@@ -1,19 +1,14 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
-
 [[package]]
 name = "astroid"
-version = "2.12.13"
+version = "2.15.5"
```

Comment:
this looks like its updated a bunch of other libraries as a byproduct

Reply:
I think the update command needs to specify eq-translations only

Reply:
When I recreated the poetry file using `poetry update eq-translations` it automatically updated the other ones

Reply:
Yeah it does look that way, or that if you do want to it would have to be specified in the toml file https://python-poetry.org/docs/cli/#update 
the update has added some new packages as well, blinker and typing-extensions. All still works and is passing so perhaps its fine

Reply:
I think that would still update it. Are you referring to "Note that this will not update versions for dependencies outside their version constraints specified in the pyproject.toml file."

Even with this, it is still going to update as long as the constraints are met.

Reply:
Exactly that yep, but then as you say it'd have to be specific version numbers